src/neuroconv/nwbconverter.py

```diff
@@ -166,9 +178,12 @@ def run_conversion(
 
         if conversion_options is None:
             conversion_options = dict()
+
         default_conversion_options = self.get_conversion_options()
         conversion_options_to_run = dict_deep_update(default_conversion_options, conversion_options)
-        self.validate_conversion_options(conversion_options=conversion_options_to_run)
+
+        if self.data_interface_classes is not None:
```

Comment:
Maybe it would be better to add the `self.data_interface_classes` in the init when data_interfaces are provided.

Reply:
Yep, probably we could use some of the new import functionality to build that on the fly so we have the workflow as similar as it was before. 

One important point to add though should do those checks by themselves though. This has been on the to-do list for a while but we have not done it. Check:
https://github.com/catalystneuro/neuroconv/issues/42

Reply:
I don't think we'll need to import. If we have objects, we can just do `obj.__class__`

Reply:
True!